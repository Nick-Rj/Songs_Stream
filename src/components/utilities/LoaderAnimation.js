import React from "react";
import HashLoader from "react-spinners/HashLoader";
import {
  FloatingLabel,
  Container,
  Form,
  Row,
  Button,
  Col,
  Modal,
  Accordion,
} from "react-bootstrap";

const overrideCSS = {
  display: "block",
  //   width: "30rem",
  //   height: "30rem",
};

const LoaderAnimation = () => {
  return (
    <div>
      <Container>
        <Row>
          <Col></Col>
          <Col>
            <HashLoader
              color={"#7ED321"}
              loading={true}
              size={150}
              css={overrideCSS}
            />
          </Col>
          <Col></Col>
        </Row>
      </Container>
    </div>
  );
};

export default LoaderAnimation;
