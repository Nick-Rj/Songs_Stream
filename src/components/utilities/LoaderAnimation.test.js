import { shallow } from "enzyme";
import { expect, it } from "@jest/globals";

import LoaderAnimation from "./LoaderAnimation";

import configureMockStore from "redux-mock-store";
import { Provider } from "react-redux";

const mockStore = configureMockStore();
const store = mockStore({
  userState: {
    loading: false,
    users: [],
    user: {},
    error: "",
    isLoggedIn: false,
  },
  songData: { loading: false, songs: [], error: "" },
  playlistData: { loading: false, playlists: [], error: "" },
});

it("rendering full DOM for LoaderAnimation Component", () => {
  const wrapper = shallow(
    <Provider store={store}>
      <LoaderAnimation />
    </Provider>
  );
  expect(wrapper).toMatchSnapshot();
});
