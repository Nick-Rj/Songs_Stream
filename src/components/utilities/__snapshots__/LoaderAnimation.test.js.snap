// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering full DOM for LoaderAnimation Component 1`] = `ShallowWrapper {}`;
